mithril-common/Cargo.toml

```
@@ -33,8 +33,12 @@ kes-summed-ed25519 = { version = "0.2.1", features = [
     "serde_enabled",
     "sk_clone_enabled",
 ] }
+minicbor = { version = "0.19", features = ["std", "half", "derive"] }
```

It looks like a newer version is available:
```suggestion
minicbor = { version = "0.20", features = ["std", "half", "derive"] }
```